device/common/include/traccc/edm/device/triplet_counter.hpp

```diff
@@ -9,37 +9,40 @@
 
 // Project include(s).
 #include "traccc/edm/container.hpp"
-#include "traccc/seeding/detail/doublet.hpp"
+#include "traccc/seeding/detail/singlet.hpp"
 
 namespace traccc::device {
 
-/// Header type for the "triplet counter container"
-///
-/// The header stores summary information about the number of triplets found in
-/// a given geometric bin.
-///
-struct triplet_counter_header {
+/// Number of triplets for one specific middle spacepoint.
+struct triplet_counter_spM {
```

Comment:
I'm now confused. :frowning: Did you change the logic with the triplets this morning? I'm not a fan of making the data layout even more complicated here...

Reply:
No, this is still the same logic we discussed. There are 2 triplet counter types, one corresponding to counts per mid-bottom doublets (as before) and another corresponding to each spacepoint middle (the new one). So you'll have 2 collections of triplet counter types linked to each other.